docs/resources/job.md

```diff
@@ -1,17 +1,13 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "qovery_job Resource - terraform-provider-qovery"
-subcategory: ""
-description: |-
-  Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
----
-
 # qovery_job (Resource)
 
 Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
 
-## Example Usage
+## Examples
```

Comment:
maybe an additional sections is too much. we could just have a "Example" section with one general example and then a paragraph:
"You can find complete examples within these repositories: ... "